govcd/api_vcd_test.go

```diff
@@ -1,4 +1,4 @@
-// +build api functional catalog vapp gateway network org query extnetwork task vm vdc system disk ALL
+// +build api functional catalog vapp gateway network org query extnetwork task vm vdc system disk unit ALL
```

Comment:
Just a question, which unit tests were added and would you try running only unit tests after this addition? I wonder how the tests cope with such request when there are so many build tags in one line.

Reply:
The "many build tags" are needed in `api_vcd_test.go` because this code needs to be activated with every test. If we don't do that, every tag call would become `-tags api catalog`, `-tags api vapp`, and so on.
But the `unit` tag should not be in this file, or it will trigger a vCD connection for tests that don't need it.

Reply:
Yeah. Moved the testing function to the same file as it was a "test for test" anyway.